guide/inheritance-wallet/recovery.md

```diff
@@ -0,0 +1,151 @@
+
+-->
+
+# Wallet recovery
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+One day, a couple of years later, Christina and David get sad news that their parents, Alice and Bob, were involved in a deadly car accident. While dealing with the inheritance proceedings, Christina and David log in to their shared family vault in their password manager. 
+
+### Accessing the backup material
+
+There, they find the PIN  to unlock the safe in their parents’ house.
+Upon opening the safe, they find the tamper-evident bags which hold the wallet recovery kit and the recovery kit password. We have covered the entire backup process [here]({{ '/guide/inheritance-wallet/backup/' | relative_url }}).
+
+### Recovering the wallet
+
+Christina and David open the bag and take the recovery PDF, which holds the wallet configuration as well as the information about all signing keys. On the first page they see the QR code with the wallet configuration. Both of them scan the code with their phones to import the wallet to our application. Once they have done that, they can see the wallet, including the balance and past activity. 
+
+{% include image-gallery.html pages = page.images_wallet-import %}
+
+However, there is a catch: they cannot spend the funds with their inheritance keys, because they have not been unlocked yet. Alice’s and Bob's last transaction was almost 5 months ago, which means that it will take more than 7 months until Christina and David can use their inheritance keys to move the funds. We have covered the wallet configuration in more detail on the [use case page]({{ '/guide/inheritance-wallet/use-case-and-scope/' | relative_url }}). Christina and David could circumvent this restriction, if they would have access to their parents’ signing devices. Unfortunately, they don’t, so they will need to wait. 
+
+### Distributing the inheritance
+
+After the timelock for the inheritance key set expires, Chrstina and David get together over dinner. They want to send half of the funds to Christina’s personal wallet and the other half to David’s wallet.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-create %}
+
+The transaction has been created and is ready to be signed. Christina goes ahead and sings it with her hardware signing device.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-christina %} 
+
+After Christina is done, David goes on to sign the transaction with his device as well.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-david %} 
+
+The first half of the inheritance has been sent to Christina's wallet. While the transaction is still being confirmed, David and Christina create and sign the second transaction, which will send the other half of the funds to David's wallet. 
+
```

Comment:
The funds are now distributed to the heirs, but the social & legal process is not finished. The lawyer may need to be informed, maybe they need to take screenshots of the fund distribution (like a receipt) for future reference. Maybe they also want to delete the wallet data, swipe the signing devices, or other stuff. Just made this up quickly, but that clean-up might be a good way to wrap up.